docs/src/site/markdown/release-notes-2-1-0.md

```diff
@@ -0,0 +1,164 @@
+    </tr>
+    <tr>
+      <td>Enable support for various non-ansi sql syntax/functionality</td>
+      <td>[TRAFODION-2180](https://issues.apache.org/jira/browse/TRAFODION-2180)</td>
+    </tr>
+    <tr>
+      <td>Use of CQD to set scratch directory locations</td>
+      <td>[TRAFODION2146](https://issues.apache.org/jira/browse/TRAFODION2146)</td>
+    </tr>
+  </table>
+</span>
+
+## Improvements
+
+<span>
+  <table>
+    <tr>
+      <th>Feature</th>
+      <th>Jira ID</th>
+    </tr>
+    <tr>
+      <td>Improve query compilation time</td>
+      <td>[TRAFODION-2137](https://issues.apache.org/jira/browse/TRAFODION-2137)</td>
+    </tr>
+    <tr>
+      <td>Improve upsert performance for table with indexes</td>
+      <td>[TRAFODION-1546](https://issues.apache.org/jira/browse/TRAFODION-1546)</td>
+    </tr>
+    <tr>
+      <td>Improve concurrency during DDL operations</td>
+      <td>[TRAFODION-2037](https://issues.apache.org/jira/browse/TRAFODION-2037)</td>
+    </tr>
+  </table>
+</span>
+
+## Install and Configuration Changes
+
+* Ambari Integration ([TRAFODION-2291](https://issues.apache.org/jira/browse/TRAFODION-2291))<br/>
+Integration with Ambari cluster manager ambari interface/integration. The Ambari integration provides support for Hortonworks Hadoop distributions, while the command-line Trafodion Installer supports Cloudera and Hortonworks Hadoop distributions, and for select vanilla Hadoop installations.
+* Python Installer ([TRAFODION-1839](https://issues.apache.org/jira/browse/TRAFODION-1839))<br/>
+Trafodion Installer Evolution The command-line installer has been replaced for the 2.1.0 release. Written in python, it replaces the legacy bash-script installer. The bash command-line installer is deprecated as of 2.1.0, but is still provided, just in case you experience any problems with the new installer. If so, please report those problems to the project team, since the legacy installer will soon be obsoleted.
+* Trafodion Configuration File ([TRAFODION-2306](https://issues.apache.org/jira/browse/TRAFODION-2306))<br/>
+Introducing a configuration file traf-site.xml specific to Trafodion similar to hbase configuration file hbase-site.xml.  This configuration file extends the properties inherited from the standard hbase-site.xml.
+* The environment variable MY_SQROOT has been renamed to TRAF_HOME.
+
+## Fixes
+
+This release contains fixes for 300+ JIRAs. Here is a [list of resolved JIRAs in Release 2.1](https://issues.apache.org/jira/issues/?jql=project%20%3D%20%22Apache%20Trafodion%22%20and%20fixVersion%20%3D%202.1-incubating%20order%20by%20updated%20desc).
+
+## Documentation Updates
+
+Several updates have been made to the SQL Reference Manual to reflect the new functions added in 2.1. Some have not been done but have JIRas outstading and will be completed by the next release. Provisioning Guide has been updated to reflect instructions for using hte new Ambari Installer or the python installation script. 
```

Comment:
"Some have not been done..." might be clearer if you said "Some functions have not been documented yet..."